yarn.lock

```diff
@@ -592,7 +592,7 @@ babel-cli@^6.23.0:
   optionalDependencies:
     chokidar "^1.6.1"
 
-babel-code-frame@^6.22.0, babel-code-frame@^6.26.0:
+babel-code-frame@^6.11.0, babel-code-frame@^6.22.0, babel-code-frame@^6.26.0:
```

Comment:
Are the changes in this file intentional? It doesn’t look like there are any new dependencies by way of `package.json` so unless you needed to upgrade an existing dependency I wouldn’t expect to see changes in this file.

Reply:
not sure why this happened